tests/functional/vault/test_misc.py

```diff
@@ -49,18 +50,19 @@ def test_credit_available_minDebtPerHarvest_larger_than_available(
     vault.initialize(
         token, gov, gov, token.symbol() + " yVault", "yv" + token.symbol(), gov
     )
-    vault.setDepositLimit(2 ** 256 - 1, {"from": gov})
+    vault.setDepositLimit(MAX_UINT256, {"from": gov})
     strategy = gov.deploy(TestStrategy, vault)
+    minDebtPerHarvest = MAX_UINT256 - 1
     vault.addStrategy(
         strategy,
         10000,  # 100% of Vault AUM
-        2 ** 256 - 2,  # minDebtPerHarvest
-        2 ** 256 - 1,  # maxDebtPerHarvest
+        minDebtPerHarvest,  # minDebtPerHarvest
```

Comment:
why is this set so high? (can it be 0)?

Reply:
In order to test the line `available < strategy_minDebtPerHarvest` `minDebtToHarvest` must be greater than `available` (it cannot be zero).

Previously I had set `minDebtToHarvest` to `MAX_UINT256 - 1` to ensure this line gets tested correctly, but I've updated the PR and now I start with `minDebtToHarvest` set to zero and update it to `available + 1`.

It should be noted that this test is specifically to test that `creditAvailable` returns zero after a liquidation when `minDebtToHarvest` is greater than `available`. We do have another unit test that covers liquidation where `minDebtPerHarvest` is zero (or less than `available`).

This is the primary assert we are testing for:
```
    minDebtPerHarvestExceedsAvailable = strategy_minDebtPerHarvest > available
    assert minDebtPerHarvestExceedsAvailable == True

    creditAvalable = vault.creditAvailable(strategy)
    assert creditAvalable == 0
```

This is the code we are adding coverage for:
![image](https://user-images.githubusercontent.com/70453354/109839378-a97e2380-7c0c-11eb-8236-2ab8073fea04.png)

Reply:
Sounds good!